Configure templates for issue creation (#3415)
rubygems/rubygems: .github/ISSUE_TEMPLATE/bundler-related-issue.md

---
name: Bundler related issue
about: This template is intended for Bundler specific related issues.
title: 'Bundler: [Please provide a title here]'

> labels: ''

Reviewer: Same as RubyGems.

assignees: ''

---

Thank you for contributing to the [rubygems](https://github.com/rubygems/rubygems) repository, and specifically to the [Bundler](https://bundler.io/) gem.

Before processing your issue please confirm that:

- [ ] You checked all the following documents and couldn't find a solution for your issue:

- [Troubleshooting common issues](https://github.com/rubygems/bundler/blob/master/doc/TROUBLESHOOTING.md)
- [Bundler documentation site](https://bundler.io/)
- [Bundler man pages](https://bundler.io/man/bundle.1.html)

> - [Bundler command line reference](https://bundler.io/v1.11/commands.html)

Reviewer 2: ```suggestion
- [Bundler command line reference](https://bundler.io/v2.0/commands.html)
```

This points the link to the latest version of the Bundler guide.

If you haven't done that, please do it before creating a new issue.

Please answer the following questions so we can process your issue as fast as possible:

> 1. Do you have a reproduction script? It would be really helpful for us if you provide one. You can click [here](https://gist.github.com/xaviershay/6207550) and [here](https://gist.github.com/xaviershay/6295889) too for a couple of examples.

Reviewer 2: Perhaps we can avoid "here" as the link text, by editing the sentence.

```suggestion
1. Do you have a reproduction script? It would be really helpful for us if you provide one. Examples: [Sample repro script for Bundler issues](https://gist.github.com/xaviershay/6207550), [Another sample repro script for Bundler issues](https://gist.github.com/xaviershay/6295889).
```

    Script URL: 

2. What are you trying to accomplish?

    Your answer:

3. What command did you run?

    Your answer:

4. What were you expecting to happen?

    Your answer:

5. What actually happened?

    Your answer:

6. Is there an exception backtrace? If so, please copy it below.

    Your answer:

7. Please run `bundle env` and paste the output below:

    Your answer:

> For more information please check [Filling Issues: a guide](https://github.com/rubygems/bundler/blob/master/doc/contributing/ISSUES.md).

Reviewer 2: An edit to the title, and the URL had changed:

```suggestion
For more information please check [Filing Issues: a guide](https://github.com/rubygems/rubygems/blob/master/bundler/doc/contributing/ISSUES.md).
```
